build.gradle

```diff
@@ -1,15 +1,18 @@
 group 'com.bardsoftware.papeeria.backend.cosmas'
 version '1.0-SNAPSHOT'
 
-apply plugin: 'java'
+

```

Comment:
Cool. Please remove extra whitespace lines. As a rule of thumb, you want to add one blank line between class/function/member variable declarations or between logically different blocks of code, but normally not more. E.g. here whitespace is not necessary.

Reply:
Done